src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -249,18 +249,10 @@ void VulnerabilityScannerFacade::start(
                             nlohmann::json dataValue;
                             dataValue["action"] = "deleteAgent";
                             dataValue["agent_info"]["agent_id"] = "000";
-                            dataValue["agent_info"]["agent_name"] = "localhost";
```

Comment:
@sebasfalcone 

Question, this name for the manager is always the same?

Reply:
I am not so sure. My agent is named with my machine name

Reply:
Fix it in [a1704b0](https://github.com/wazuh/wazuh/commit/a1704b0b954cb1fd18ec12c2de55bd3c87d9872c)